pom.xml

```diff
@@ -405,6 +405,30 @@
                  <goal>verify</goal>
               </goals>
             </execution>
+            <execution>
+              <id>SignOffTests</id>
+              <configuration>
+                <reuseForks>false</reuseForks>
+                <argLine>@{jacocoArgLine} -enableassertions -Xmx3000m -Djava.security.egd=file:/dev/./urandom "-Djava.library.path=${hadoop.library.path}${path.separator}${java.library.path}" -XX:+HeapDumpOnOutOfMemoryError -XX:HeapDumpPath=./target/ -XX:NewRatio=4 -XX:SurvivorRatio=8 -XX:+UseCompressedOops -XX:+UseConcMarkSweepGC -XX:+DisableExplicitGC -XX:+UseCMSInitiatingOccupancyOnly -XX:+CMSClassUnloadingEnabled -XX:+CMSScavengeBeforeRemark -XX:CMSInitiatingOccupancyFraction=68</argLine>
+                <groups>org.apache.phoenix.end2end.SupportsDistributedClusterTest</groups>
+              </configuration>
+              <goals>
+                <goal>integration-test</goal>
+                <goal>verify</goal>
+              </goals>
+            </execution>
+            <execution>
+              <id>FailedSignOffTests</id>
+              <configuration>
+                <reuseForks>false</reuseForks>
+                <argLine>@{jacocoArgLine} -enableassertions -Xmx3000m -Djava.security.egd=file:/dev/./urandom "-Djava.library.path=${hadoop.library.path}${path.separator}${java.library.path}" -XX:+HeapDumpOnOutOfMemoryError -XX:HeapDumpPath=./target/ -XX:NewRatio=4 -XX:SurvivorRatio=8 -XX:+UseCompressedOops -XX:+UseConcMarkSweepGC -XX:+DisableExplicitGC -XX:+UseCMSInitiatingOccupancyOnly -XX:+CMSClassUnloadingEnabled -XX:+CMSScavengeBeforeRemark -XX:CMSInitiatingOccupancyFraction=68</argLine>
```

Comment:
I have reorganized the options handling some time ago.
Please use the new system.

Reply:
What is the value of this ?

De we really need a new category for the tests that cannot be run a distributed cluster ?

Reply:
This is really meant for the next iteration when we try to resolve the failed test. But I guess we can consider all non-tagged tests as failed ones. 
FYI @Divneet18